packages/schema/src/types/config.ts

```diff
@@ -135,7 +135,10 @@ export interface RuntimeConfig extends RuntimeConfigNamespace {
 }
 
 // -- App Config --
-export interface AppConfigInput extends Record<string, any> {
+
+export interface _AppConfig extends Record<string, any> { }
```

Comment:
I'm thinking to rename it back to `CustomAppConfig` (we might use `_AppConfig` for merged type of `useAppConfig` later similar to plugins). Wyt @danielroe ?

Reply:
`CustomAppConfig` sounds better, and more understandable, for sure.